src/ocean/task/internal/SpecializedPools.d

```diff
@@ -0,0 +1,178 @@
+/*******************************************************************************
+
+    Implements mapping between specific task `TypeInfo` and matching
+    `FiberPoolEager` instances intended to serve all tasks of such type.
+
+    Copyright: Copyright (c) 2017 sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.task.internal.SpecializedPools;
+
+import ocean.meta.types.Qualifiers;
+import ocean.task.internal.FiberPoolEager;
+import ocean.task.Task;
+import ocean.core.Verify;
+import ocean.core.Enforce;
+import ocean.core.Optional;
```

Comment:
Should the imports be sorted?

Reply:
No, there is no particular import order requirement for our libs/apps.

Reply:
fair enough